ocs_ci/ocs/must_gather/must_gather.py

```
@@ -61,6 +61,19 @@ def collect_must_gather(self, ocs_flags=None, mg_options=None):
             dir_name=temp_folder, ocp=False, ocs_flags=ocs_flags, mg_options=mg_options
         )
         self.root = temp_folder + "_ocs_logs"
+        root_exists = os.path.isdir(self.root) if self.root else False
+        root_contents = []
+        if root_exists:
+            for dirpath, dirnames, filenames in os.walk(self.root):
```

Ruff (0.15.20)

[warning] Loop control variable `dirnames` not used within loop body

Rename unused `dirnames` to `_dirnames`

(B007)